src/com/sun/ts/tests/servlet/api/jakarta_servlet/genericservlet/ServletErrorPage.java

```diff
@@ -39,7 +39,7 @@
 
   private static final String STATUS_CODE = "jakarta.servlet.error.status_code";
 
-  private static final String EXCEPTION_TYPE = "javax.serlvet.error.exception_type";
+  private static final String EXCEPTION_TYPE = "javax.servlet.error.exception_type";
```

Comment:
weird this one #277 should have been merged first!
this change just introduce more issue..

Reply:
It will be good to have the other serlvet typos also fixed (in ts.jte comments + doc files).  As well as having your change as well soon.